core/src/components/item/test/disabled/index.html

```diff
@@ -134,8 +128,7 @@
 
           <ion-item>
             <ion-checkbox slot="start" aria-label="Checkbox"></ion-checkbox>
-            <ion-label>Checkbox + Range</ion-label>
-            <ion-range disabled value="45" aria-label="Range"></ion-range>
```

Comment:
We should probably just remove this entirely since it will be removed with checkbox anyway and it doesn't check what it was testing anymore